SDWebImage/SDWebImageImageIOCoder.m

```diff
@@ -484,9 +497,9 @@ + (BOOL)canEncodeToHEICFormat {
             canEncode = NO;
         } else {
             // Can encode to HEIC
-            CFRelease(imageDestination);
             canEncode = YES;
         }
+        CFRelease(imageDestination);
```

Comment:
...Wait, why you want to call `CFRelease` on a `NULL` ?

Reply:
My misstake,ignore it.